web-common/src/components/data-graphic/marks/types.ts

```diff
@@ -1,7 +1,7 @@
 export interface Point {
   x: number;
   y: number;
-  value: string;
+  value?: string;
```

Comment:
this is already treated as optional, there are guards against it being undefined in web-common/src/components/data-graphic/marks/MultiMetricMouseoverLabel.svelte line 204